Fix typos and grammar in tags (#1836)
python-discord/bot: bot/resources/tags/traceback.md

Please provide the full traceback for your exception in order to help us identify your issue.

A full traceback could look like:
```py
Traceback (most recent call last):
    File "tiny", line 3, in
        do_something()
    File "tiny", line 2, in do_something
        a = 6 / b
ZeroDivisionError: integer division or modulo by zero
```
The best way to read your traceback is bottom to top.

• Identify the exception raised (e.g. `ZeroDivisionError`)  
• Make note of the line number (in this case 2), and navigate there in your program.  

> • Try to understand why the error occurred (in this case because `b` must be `0`).

Reviewer: ```suggestion
• Try to understand why the error occurred (in this case probably because `b` is `0`).
```

Author: Even "most likely" isn't correct. We know categorically that `b` *is* `0` from this error. There's no other way for the error to be raised.

I agree that `is` flows better than `must be` though so will change that.

Also going to update the error message since apparently I forgot to do that.

Reviewer: Yup sounds good
